src/User/Handlers.hs

```diff
@@ -22,5 +24,18 @@ handleUserAdd = do
   sd <- liftIO getSystemDay
   (view, muf) <- runForm "user_form" $ userForm sd
   case muf of
-    Just uf -> insertUser uf >> redirect "/users"
-    Nothing -> heistLocal (bindDigestiveSplices view) $ render "users_add" 
\ No newline at end of file
+    Just uf -> do
+      r <- insertUser uf
+      bool (writeBS "Something went wrong while adding the user.") 
+        (redirect "/users") (r>0)
+    Nothing -> heistLocal (bindDigestiveSplices view) $ render "users_add" 
+
+handleUserDelete :: AppHandler
+handleUserDelete = do
+  key <- getParam "id"
+  r <- deleteUser 
+        (maybe 
+          (error "Could not parse id parameter") 
+          (\bs -> read $ BS.unpack bs :: Int) key)
```

Comment:
@kinorius123 Can you replace `\bs -> read $ BS.unpack bs` with `read . BS.unpack`?
@kinorius123 Also, could you please use a more explicit name than `r`?